datadog_checks_downloader/tests/test_downloader.py

```diff
@@ -140,18 +437,22 @@ def get_all_integrations_metadata():
     return dict(results)
 
 
-def test_downloader():
+@pytest.mark.online
+def test_downloader(verify_all_integrations):
+    if not verify_all_integrations:
```

Comment:
With this addition, now we are not testing all integrations on every run in the CI. Can we revert this?

Reply:
The problem was that this test takes a lot of time to complete. CI checks timeout in these cases.